src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -494,6 +493,10 @@
     <Error Condition="!Exists('..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets'))" />
   </Target>
   <Import Project="..\..\packages\Microsoft.Azure.DocumentDB.1.9.2\build\Microsoft.Azure.DocumentDB.targets" Condition="Exists('..\..\packages\Microsoft.Azure.DocumentDB.1.9.2\build\Microsoft.Azure.DocumentDB.targets')" />
+  <Target Name="CopyAzureFunctionsToNodeModules" AfterTargets="Build">
```

Comment:
Agree with all the points you've made, a first class module is heavier than necessary and I was pretty sure this wouldn't hit npm.  

Correct me if I'm wrong, but it doesn't seem possible to `require` an embedded resource.  Linked resource may be possible, but are there any benefits to adding the javascript files as a linked resource?

I stuck with `node_modules\azure-functions\index.js` for convention's sake, but deploying a `functions\functions.js` would work just as well.